packages/react/src/components/cookieConsent/buttons/Buttons.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import { CookieConsentActionListener } from '../types';
+import { Button } from '../../button/Button';
+import styles from '../CookieConsent.module.scss';
+
+export type Props = {
+  onClick: CookieConsentActionListener;
+};
+
+function Buttons({ onClick }: Props): React.ReactElement {
+  return (
+    <div className={styles.buttons}>
+      <Button
+        variant="primary"
+        onClick={() => {
+          onClick('approveAll');
+        }}
+        data-testid="cookie-consent-approve-all-button"
+      >
+        Hyväksy kaikki evästeet
+      </Button>
+      <Button
+        variant="secondary"
+        onClick={() => {
+          onClick('approveRequired');
+        }}
+        data-testid="cookie-consent-approve-required-button"
+        aria-label="Hyväksy pakolliset evästeet"
+      >
+        Pakolliset evästeet
+      </Button>
+      <Button
+        variant="secondary"
+        onClick={() => {
+          onClick('showDetails');
+        }}
+        data-testid="cookie-consent-read-more-button"
+      >
+        Asetukset
```

Comment:
Should the settings button be hidden when the user is on the cookie settings view?

Reply:
Outdated.